.azure-pipelines/generation-templates/php-run-tests-beta.yml

```diff
@@ -0,0 +1,17 @@
+# https://docs.microsoft.com/en-us/azure/devops/pipelines/ecosystems/php?view=azure-devops
+
+steps:
```

Comment:
reuse https://github.com/microsoftgraph/MSGraph-SDK-Code-Generator/blob/dev/.azure-pipelines/generation-templates/php-run-tests.yml and make the working directory a template parameter instead of duplicating